kedro/framework/cli/pipeline.py

```diff
@@ -101,26 +101,26 @@ def create_pipeline(
     env = env or "base"
     if not skip_config and not (project_conf_path / env).exists():
         raise KedroCliError(
-            f"Unable to locate environment `{env}`. "
+            f"Unable to locate environment '{env}'. "
             f"Make sure it exists in the project configuration."
         )
 
     result_path = _create_pipeline(name, package_dir / "pipelines")
     _copy_pipeline_tests(name, result_path, package_dir)
     _copy_pipeline_configs(result_path, project_conf_path, skip_config, env=env)
-    click.secho(f"\nPipeline `{name}` was successfully created.\n", fg="green")
+    click.secho(f"\nPipeline '{name}' was successfully created.\n", fg="green")
 
     click.secho(
-        f"To be able to run the pipeline `{name}`, you will need to add it "
-        f"to `register_pipelines()` in `{package_dir / 'pipeline_registry.py'}`.",
+        f"To be able to run the pipeline '{name}', you will need to add it "
+        f"""to 'register_pipelines()' in "{package_dir / 'pipeline_registry.py'}".""",
```

Comment:
```suggestion
        f"""to 'register_pipelines()' in '{package_dir / "pipeline_registry.py"}'.""",
```
since the final string that is rendered will be the outer quote marks (outside the `{}`), which you're more commonly doing with single quote marks 🙂